src/libs/map.ts

```diff
@@ -0,0 +1,9 @@
+export const getIsInclude = <K, V>(map: Map<K, V>, key: K) => !!map.get(key)
+
+export const mapToArray = <K, V>(map: Map<K, V>) => Array.from(map.keys())
+
+export const arrayToMap = <T>(array: T[]) => {
```

Comment:
[MUST] Like `mapToKeyArray`, `arrayToMap` does not match the expected behavior from the name. I think the name `flatArrayToMap` is appropriate.